qrexec-lib/unicode-allowlist-table.c

```diff
@@ -1,12 +1,12 @@
-    case 0x000020 ... 0x00007e:
-    case 0x0000a1:
+    case 0x000020 ... 0x00005d:
+    case 0x00005f:
+    case 0x000061 ... 0x00007e:
```

Comment:
So, now "^" and "`" are excluded, is that intentional? The latter might be a good idea, but the commit message talks only about allowing some more...

Reply:
Nope, that’s a bug.  Maybe I should create a file with every allowed character and then add a test to make sure they stay allowed!